corehq/apps/cleanup/tasks.py

```diff
@@ -245,18 +239,6 @@ def check_for_ucr_tables_without_existing_domain():
         mail_admins_async.delay('All UCR tables belong to valid domains', '')
 
 
-@periodic_task(run_every=crontab(minute=0, hour=16), queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'))
-def delete_web_user():
-    if settings.SERVER_ENVIRONMENT == 'production':
-        for username in [
-            'create_growth' + '@' + 'outlook.com',
-            'growth_analytics' + '@' + 'outlook.com',
-        ]:
-            web_user = WebUser.get_by_username(username)
-            if web_user:
-                web_user.delete(None, deleted_by=SYSTEM_USER_ID, deleted_via=__name__ + '.delete_web_user')
```

Comment:
confirmed good to remove https://github.com/dimagi/commcare-hq/pull/29901#issuecomment-865830527

Reply:
🏆  for following through on this, thank you